streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamSessionWindowAggregate.java

```diff
@@ -35,22 +35,22 @@
 import java.util.ArrayList;
 import java.util.List;
 
-class KStreamSessionWindowAggregate<K, V, T> implements KStreamAggProcessorSupplier<K, Windowed<K>, V, T> {
+class KStreamSessionWindowAggregate<K, V, Agg> implements KStreamAggProcessorSupplier<K, Windowed<K>, V, Agg> {
```

Comment:
I think "K" = "key type" and "V" = "value type" are pretty intuitive. It took me a little while to figure out that "T" = "aggregator type".

Is it ok to give this type parameter a more obvious name?

Reply:
Should be fine.